src/Microsoft.DotNet.Interactive/Kernel.cs

```diff
@@ -317,6 +338,24 @@ public void RegisterCommandType<TCommand>()
             return context.Result;
         }
 
+        private async Task RunOnFastPath(CancellationToken cancellationToken, KernelInvocationContext context,
```

Comment:
`CancellationToken` is typically the last parameter.

Reply:
fixing now